java/layer-wrapper/scripts/otel-handler

```diff
@@ -9,4 +9,4 @@ fi
 
 export OTEL_INSTRUMENTATION_AWS_LAMBDA_FLUSH_TIMEOUT=10000
 
-exec "$@"
+exec "${@//$OTEL_INSTRUMENTATION_AWS_LAMBDA_HANDLER/$_HANDLER}"
```

Comment:
I think these would benefit with a brief comment explaining the approach with maybe some examples (before/after).

Reply:
Good idea, will do